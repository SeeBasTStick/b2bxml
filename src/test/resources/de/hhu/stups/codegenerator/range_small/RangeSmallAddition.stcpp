int main() {
    RangeSmall range;
    range.simulate();
    cout << range.getSet().card();
    return 0;
}