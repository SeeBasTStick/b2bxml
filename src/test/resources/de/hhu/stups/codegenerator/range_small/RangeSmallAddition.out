public static void main(String[] args) {
    RangeSmall range = new RangeSmall();
    range.simulate();
    System.out.println(range.set);
}